From 61746b75a7b10115633df2f9dce075c0bbdf161e Mon Sep 17 00:00:00 2001
From: Jan Cholasta <jcholast@redhat.com>
Date: Tue, 9 Aug 2016 14:12:50 +0200
Subject: [PATCH] Revert "spec: add conflict with bind-chroot to
 freeipa-server-dns"

Remove the conflict, as bind-chroot caused issue only on systems with older
bind and bind-chroot - e.g. RHEL 6.

This reverts commit 3ab63fa6ba60947b1452c2108c4cf7637f4aacdb.

https://fedorahosted.org/freeipa/ticket/5696

Reviewed-By: Petr Spacek <pspacek@redhat.com>
---
 freeipa.spec.in | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/freeipa.spec.in b/freeipa.spec.in
index e1d98a44d04804a6bb2d4b17206aa7c280d64eb9..77fca24ef0e1e244e53d203a228931b4a444fb5a 100644
--- a/freeipa.spec.in
+++ b/freeipa.spec.in
@@ -274,9 +274,6 @@ Obsoletes: %{alt_name}-server-dns < %{version}
 # upgrade path from monolithic -server to -server + -server-dns
 Obsoletes: %{name}-server <= 4.2.0
 
-# FreeIPA does not support running integrated BIND in chroot jail
-Conflicts: bind-chroot
-
 %description server-dns
 IPA integrated DNS server with support for automatic DNSSEC signing.
 Integrated DNS server is BIND 9. OpenDNSSEC provides key management.
-- 
2.7.4

